dev/fattest.simplicity/build.gradle

```
@@ -20,73 +43,43 @@ task assembleBootstrap(type: Copy) {
     filter(ReplaceTokens, tokens: [JAVA_HOME: javaHome])
 }
 
+task assembleBinaryDependencyJar(type:Copy) {
+	from jar
+	into new File(buildDir, 'lib')
+	exclude 'buildfiles'
```

```suggestion
	include 'fattest.simplicity.jar'
```
Would it be better to just do an include instead of having to update to have extra excludes if something else get added we don't want to be included?  If we add an extra jar (less likely) that we want to get included that is fine.  Or does doing that end up breaking what you are actually trying to fix?

---

Either way works. :)